report_thesis/src/sections/background.tex

```diff
@@ -93,7 +93,24 @@ \subsubsection{Principal Component Analysis (PCA)}
 where $\mathbf{V}_n$ is the matrix that only contains the top $n$ eigenvectors.
 
 \subsubsection{Kernel PCA}
-% \citet{scholkopftKPCA}
+The description of the \gls{kernel-pca} algorithm is based on \citet{learningwithkernels}.
+\gls{kernel-pca} is an extension of the traditional \gls{pca} that is used to handle nonlinear relationships among data points.
+The core idea behind \gls{kernel-pca} is to map the data into a higher-dimensional space using a kernel function, known as the \textit{kernel trick}. 
+This mapping allows for the linear separation of data points in the higher-dimensional space, even when the data is not linearly separable in the original space.
+
+The core element of \gls{kernel-pca} is to obtain the covariance matrix in the higher-dimensional space. 
+Mathematically, this is given by:
+$$
+	\mathbf{C} = \frac{1}{m} \sum_{j=1}^{m} \Phi(\mathbf{x}_j) \Phi(\mathbf{x}_j)^\top
+$$
+where $m$ is the number of data points and $\Phi(\mathbf{x}_j)$ represents the mapping of the data point $\mathbf{x}_j$ into the higher-dimensional feature space. 
+The term $\frac{1}{m}$ normalizes the sum, ensuring the covariance matrix represents the average pairwise relationships between data points.
+
+However, in practice, computing the covariance matrix is typically not feasible for large datasets, as computing the mapping function $\Phi(\mathbf{x}_j$ for each data point is expensive and often impractical due to the high dimensionality of the feature space. 
+Instead, Kernel PCA leverages the kernel trick, which allows us to compute the similarity between data points directly in the original space using a kernel function $k(\mathbf{x}_i, \mathbf{x}_j)$. 
+This kernel function implicitly computes the dot product $\Phi(\mathbf{x}_i)^\top \Phi(\mathbf{x}_j)$ in the higher-dimensional feature space without explicitly performing the mapping. 
+By constructing the kernel matrix \(\mathbf{K}\) using these pairwise similarities, Kernel PCA can perform eigenvalue decomposition to obtain the principal components in the feature space, in the same fashion as regular \gls{pca} described in Section~\ref{subsec:pca}.
+However, the eigenvalue decomposition is performed on the kernel matrix $\mathbf{K}$ rather than the covariance matrix $\mathbf{C}$.
```

Comment:
```suggestion
However, in Kernel PCA, the eigenvalue decomposition is performed on the kernel matrix \(\mathbf{K}\) rather than the covariance matrix \(\mathbf{C}\).
```